subsys/bluetooth/host/gatt.c

```diff
@@ -3475,10 +3475,17 @@ bool bt_gatt_is_subscribed(struct bt_conn *conn,
 
 	/* Check if attribute is a characteristic declaration */
 	if (!bt_uuid_cmp(attr->uuid, BT_UUID_GATT_CHRC)) {
-		struct bt_gatt_chrc *chrc = attr->user_data;
+		uint8_t properties;
+		ssize_t len;
+
+		/* The charactestic properties is the first byte of the attribute value */
+		len = attr->read(NULL, attr, &properties, 1, 0);
+		if (len < 0) {
```

Comment:
```suggestion
		if (len < 1) {
```

Reply:
Does the `LOG_ERR("Failed to read attribute: %zd", len);` match the case where len == 0? Or should we have different log messages for failure to read and invalid read length?

Reply:
I added a separate check for invalid read length